core/iwasm/libraries/lib-pthread/lib_pthread_wrapper.c

```diff
@@ -5,6 +5,7 @@
 
 #include "bh_common.h"
 #include "bh_log.h"
+#include "platform_api_extension.h"
```

Comment:
Should include"bh_platform.h" instead, "platform_api_extension.h" is an internal file of core/shared and is included in "bh_platform.h", normally we don't include it.

Reply:
Sure, this was added by clangd in my editor, I'll remove it.